### ISSUE TITLE
can not reproduce DNA result on citation benchmark 

### ISSUE THREAD

**Issue by shaybek** [rusty1s/pytorch_geometric#645]
## ❓ Questions & Help

Hello there,
I'm trying to reproduce DNA result on the citation benchmark but got a low accuracy( 0.211 ± 0.083). Here is the code. 
``` python
parser = argparse.ArgumentParser()
parser.add_argument('--dataset', type=str, default="cora")
parser.add_argument('--random_splits', type=bool, default=True)
parser.add_argument('--runs', type=int, default=10)
parser.add_argument('--epochs', type=int, default=200)
parser.add_argument('--lr', type=float, default=0.005)
parser.add_argument('--weight_decay', type=float, default=0.0005)
parser.add_argument('--early_stopping', type=int, default=10)
parser.add_argument('--hidden_channels', type=int, default=128)
parser.add_argument('--num_layers', type=int, default=5)
parser.add_argument('--dropout', type=float, default=0.5)
parser.add_argument('--normalize_features', type=bool, default=True)
parser.add_argument('--heads', type=int, default=8)
parser.add_argument('--groups', type=int, default=16)
args = parser.parse_args('')
class NetDNA(torch.nn.Module):
    def __init__(self, dataset):
        super(NetDNA, self).__init__()
        self.hidden_channels = args.hidden_channels
        self.lin1 = nn.Linear(dataset.num_features, self.hidden_channels)
        self.convs = nn.ModuleList()
        
        for i in range(args.num_layers):
          self.convs.append(DNAConv(self.hidden_channels,args.heads, args.groups, dropout=0.8, cached=True))
          
        self.lin2 = torch.nn.Linear(self.hidden_channels, dataset.num_classes)
        
        
    def reset_parameters(self):
        self.lin1.reset_parameters()
        for conv in self.convs:
            conv.reset_parameters()
        self.lin2.reset_parameters()

    def forward(self, data):
        x, edge_index = data.x, data.edge_index
        x = F.relu(self.lin1(x))
        x = F.dropout(x, p=0.5, training=self.training)
        x_all = x.view(-1, 1, self.hidden_channels)
        for conv in self.convs:
            x = F.relu(conv(x_all, edge_index))
            x = x.view(-1, 1, self.hidden_channels)
            x_all = torch.cat([x_all, x], dim=1)
        x = x_all[:, -1]
        x = F.dropout(x, p=0.5, training=self.training)
        x = self.lin2(x)
        return torch.log_softmax(x, dim=1)


dataset = get_planetoid_dataset(args.dataset, args.normalize_features)
permute_masks = random_planetoid_splits if args.random_splits else None
dna=NetDNA(dataset)
run(dataset, dna, args.runs, args.epochs, args.lr, args.weight_decay,
    args.early_stopping, permute_masks)
```

**Comment by rusty1s** [comment]
I haven't rerun your code, but this might be due to early stopping. The `DNAConv` needs some time to get going, so increasing this might likely solve this issue. Note that I use a different random split in the paper than the one you use from the benchmark folder.
